image/mkimage.sh

```diff
@@ -21,28 +23,24 @@ else
 fi
 
 # check for tools
-if ! command -v mdir --help >/dev/null 2>&1; then
+if ! which mdir > /dev/null 2>&1; then
```

Comment:
`command` is posix while `which` isn't. does OpenBSD really lack it ? alternatively, `type` can be used.

Reply:
I checked it again, this is caused  by "mdir **--help**" issue.
In the first place, is there any argument required for mdir when existing check by 'command -v'？